### PR TITLE
Style changes and add/change some checks.

### PR DESCRIPTION
**Pull request by ebezzam**
Thanks for sending a pull request (PR), we really appreciate that! Before hitting the submit button, we'd be really glad if you could make sure all the following points have been cleared.

Please also refer to the doc on [contributing](https://pyroomacoustics.readthedocs.io/en/pypi-release/contributing.html) for more details. Even if you can't check all the boxes below, do not hesitate to go ahead with the PR and ask for help there.

- [ ] Are there docstrings ? Do they follow the [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard) style ?
- [ ] Have you run the tests by doing `nosetests` or `py.test` at the root of the repo ?
- [ ] Have you checked that the doc builds properly and that any new file has been added to the repo ? How to do that is covered in the [documentation](https://pyroomacoustics.readthedocs.io/en/pypi-release/contributing.html#documentation).
- [ ] Is there a unit test for the proposed code modification ? If the PR addresses an issue, the test should make sure the issue is fixed.
- [ ] Last but not least, did you document the proposed change in the CHANGELOG file ? It should go under "Unreleased".

Happy PR :smiley:

### CODE REVIEW

File: pyroomacoustics/room.py [hunk 1]
```diff
@@ -1743,7 +1744,7 @@ class ShoeBox(Room):
 
     def __init__(self,
                  p,
-                 fs=8000,
+                 fs=16000,
```
Comment (fakufaku) [inline pyroomacoustics/room.py]: I don't think we should change the defaults. This will break code downstream.

File: pyroomacoustics/room.py [hunk 2]
```diff
@@ -1808,28 +1809,32 @@ def __init__(self,
...
+                        '`absorption` parameter is ignored.',
                         DeprecationWarning,
                         )
 
             if isinstance(materials, Material):
                 materials = dict(zip(self.wall_names, [materials] * n_walls))
-            elif not isinstance(materials, dict):
-                raise ValueError('Materials must be a string, Material object, or dictionary')
+            else:
+                raise ValueError('`materials` must be a `Material` object.')
```
Comment (fakufaku) [inline pyroomacoustics/room.py]: Why not keep the dictionary ? I think it can be good if we want to set different materials for walls.

Comment (ebezzam) [reply pyroomacoustics/room.py]: ah sorry, I thought it was a "leftover" like the string! but now I see later on you were creating a dict for all cases to pass to libroom. I'll put it back and also add some docstring :)

File: pyroomacoustics/room.py (hunk 1 above)
Comment (ebezzam) [reply pyroomacoustics/room.py]: agreed

**Pull request merged**